LongestPalindrome.java

```diff
@@ -0,0 +1,23 @@
+// Time complexity -- O(n)
+// Space complexity  -- O(n)
```

Comment:
Space will be O(1) as the maximum characters in set can be 26 all lower case chars which will be constant